frontend/src/features/admin-form/create/workflow/components/WorkflowContent/EditStepBlock/EditStepBlock.stories.tsx

```diff
@@ -129,6 +129,61 @@ export const Step2Empty = {
   },
 }
 
+export const Step2FixedEmailDefault = {
+  // due to the double registration of 'workflow_type' there would be a weird interaction
```

Comment:
This is still weird imo. But couldn't find good and easy solution.

Reply:
im not really sure what it means to `manually select the field again`. I'm currently seeing an interaction test where we are 
- expecting save step button not to be disabled (perhaps for general coverage?) 
- clicks on the fixed emails (I can understand that this is to set the state of the component into the fixed emails toggle clicked) state for visual review 

can't seem to find the code that manually selects the field.

Reply:
> await userEvent.click(await canvas.getByText('Fixed email(s)'))

is the code that manually selects the field

Reply:
I get it now, instead of setting it using the args, we need to use interaction tests to click on the static email radio manually